### PR TITLE
[MM-T666] /groupmsg error if messaging more than 7 users

### PR DESCRIPTION
**Pull request by clarmso**
#### Summary

This tests ensures that the group messages cannot be sent to more than 7 users.

#### Ticket Link

https://automation-test-cases.vercel.app/test/MM-T666

#### Related Pull Requests

- Has server changes N/A
- Has redux changes N/A
- Has mobile changes N/A

#### Screenshots

Screen Shot 2020-08-03 at 9 09 57 PM

### CODE REVIEW

**Comment by hanzei** [comment]
@clarmso Heads up that your PR has a merge conflict.

File: e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js [hunk 1]
```diff
@@ -11,10 +11,12 @@
 // Group: @integrations
 
 import * as TIMEOUTS from '../../../fixtures/timeouts';
+import * as MESSAGES from '../../../fixtures/messages';
 
 describe('I18456 Built-in slash commands: common', () => {
     let user1;
     let user2;
+    let userGroup;
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:19]: Should directly assign empty array here.
```suggestion
    let userGroup = [];
```

File: e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js [hunk 2]
```diff
@@ -27,6 +29,16 @@ describe('I18456 Built-in slash commands: common', () => {
...
+
+            const prefixes = ['hello', 'hello', 'hello', 'hello', 'hello', 'hello', 'hello', 'hello'];
+            userGroup = [];
+            prefixes.forEach((prefix) => {
+                cy.apiCreateUser({prefix}).then(({user: otherUser}) => {
+                    const newUser = otherUser;
+                    cy.apiAddUserToTeam(team.id, newUser.id);
+                    userGroup.push(newUser);
+                });
+            });
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:41]: I can't see where the prefix is used during test step/verification. Can simply be,
```suggestion
            Cypress._.times(8, () => {
                cy.apiCreateUser().then(({user: otherUser}) => {
                    cy.apiAddUserToTeam(team.id, otherUser.id);
                    userGroup.push(otherUser);
                });
            });
```

Comment (clarmso) [reply e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:41]: 👍 Great! I just looking for a way to create a specific number of users without using a for-loop.

**Comment by josephbaylon** [comment]
@clarmso Sorry didn't notice @saturninoabril already pointed it out.

**Comment by clarmso** [comment]
Hold on...I haven't forgotten to resolve the conflicts.

**Pull request merged**